.env

```diff
@@ -15,5 +15,5 @@ APP_SECRET=d28750a79bbdda939b77d930e53a3ae2
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
 # Configure your db driver and server_version in config/packages/doctrine.yaml
-DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name
+DATABASE_URL=mysql://root:123@127.0.0.1:3306/db_symfony4
```

Comment:
please remove .env file from git, instead, you should add .env.dist file with standard credentials rather than your local one.